### ISSUE TITLE
Navigation is confusing-to-nonexistent with devolved parliaments

### ISSUE THREAD

**Issue by handelaar** [mysociety/theyworkforyou#1721]
Congrats on Senedd :)

I was told today by local about the launch (so the PR also worked).  And went to theyworkforyou.com, and spent well over two minutes trying to find it.

- Apart from the (temporary) launch banner there are no links on the home page *at all* to either Wales or Scotland in the primary nav
- There's nothing in the site footer either
- The actual selector is basically not visible and certainly doesn't look like a selector even if you can see it

Then once we're at /senedd/ 

- Headline item is to a random debate.  There is no link to the debate that this random speech was lifted from, nor to the random speech itself -- only one to the speaker page ![image](https://github.com/mysociety/theyworkforyou/assets/142903/b545742e-2c2d-40ec-bcac-f8c99e531f6b)
- There is no browse or navigation to debates from here at all.  There is a minuscule link to /senedd/?more=1 with the link text "Senedd" - that's... obscure.  There we will finally find a link to calendar browsing
- (These same issues afflict /scotland/ )

Basically the site no longer has any navigation or click-around-ability for devolved parliaments; it's all been removed at some point in favour of navigation by searchbox only.  This has usability roadblocks, including the requirement to know what to search for in advance before being shown anything useful,  that are severe and of a nature people here are more than familiar with already.

Apols, but it's *bad*, and I'm surprised.

**Comment by dracos** [comment]
The lack of heading on the 'random debate' sometimes should now be fixed.

**Comment by handelaar** [comment]
> "Apart from the (temporary) launch banner there are no links on the home page at all to either Wales or Scotland in the primary nav" - I fully agree this is not ideal, (perhaps this is what you mean by selector) you can click the dropdown "UK" and there are links to all the other entities there. This has grown into this over time.

I found the dropdown eventually, yes, but it's *extremely* obfuscated (a microscopic arrow in pale grey on white, in a counterintuitive location).  Adding "Scotland/Wales/NI/London" to that primary nav with current breakpoints would likely not fit, but _only barely_ and I think the loss of navigability is a very high price to pay for the lack of a solution there.  And in the collapsed/mobile menu there's no excuse at all for their omission, imho.  Plus:  no content nav in footer?

Basically my point is:  I *run* one of these, and I was looking for it, and *_*even I*_* couldn't find it... 

> There are also other recent debates lower down you didn't mention. I agree the homepages themselves could do with improvement, the link to the more browsing page should be easier/more prominent, and perhaps the calendar could be shown on the RHS of the main page, 

There's a great deal of needlessly-tall header and promo stuff nobody needs there, and no "browse" nav.  

I mean I get this (I have previously failed on KS to think of something else to put on these index pages, same as everybody else has).  But... calendar for sure, surely.  Whatever happened to listing out all the headings for the most recent sitting day instead of taking the "random headings" space-filler and using it *more*?  That certainly seems like a more "discoverable" interface, Again: to me.  Perhaps there's counter-research somewhere to justify this

**Comment by ajparsons** [comment]
The big problem of navigation from the homepage has been addressed through https://github.com/mysociety/theyworkforyou/issues/1707. 

I've made a set of tidy-up changes to devolved parliaments, adding a clearer link to the 'recent debates' view, and in favour of emphasising the search. Our big thing is having everything from the different parliaments in one place for search, member profiles and alerts. We could do more here, and it might make sense to as part of a future project, but is beyond what we wanted to do with this project. 

The 'random debate' is fallback behaviour for the lack of a featured debate. At a later time we might replace this with the announcement space used on the home page. But de-emphasised for the moment.
